eng/Configurations.props

```diff
@@ -20,7 +20,7 @@
   <PropertyGroup>
     <NETCoreAppCurrentVersion>5.0</NETCoreAppCurrentVersion>
     <NetCoreAppCurrentTargetFrameworkMoniker>.NETCoreApp,Version=v$(NETCoreAppCurrentVersion)</NetCoreAppCurrentTargetFrameworkMoniker>
-    <NetCoreAppCurrent>netcoreapp$(NETCoreAppCurrentVersion)</NetCoreAppCurrent>
+    <NetCoreAppCurrent>net$(NETCoreAppCurrentVersion)</NetCoreAppCurrent>
```

Comment:
This change seems suspicious. The only current known target framework for which this property is valid is `net5.0`. Hence this seems like configuration for something that only has one right answer. This can't for example ever be `netcoreapp3.1`. Is the intent here to provide flexibility for future versions?

Reply:
> Is the intent here to provide flexibility for future versions?

Correct. By incrementing the `NETCoreAppCurrentVersion` we can rev when switch to net6.0 without updating all the consumers of this property.